### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by jdubs11** [ask/mode#71]
Is this the new fork?
https://github.com/faust-streaming/mode/

**Comment by wbarnha** [comment]
Yeah, Ask Solem sadly passed away so there's noone to maintain the original projects.
